src/widgets/AutocompleteSelector.js

```diff
@@ -93,12 +93,11 @@ export class AutocompleteSelector extends React.PureComponent {
       <View style={localStyles.container}>
         <SearchBar
           autoCapitalize="none"
-          autoCorrect={false}
+          autoCorrect={true}
           autoFocus
-          color="white"
-          onChange={text => this.setState({ queryText: text })}
           placeholder={placeholderText}
-          placeholderTextColor="white"
+          debouncedCallback
+          onChangeText={text => this.setState({ queryText: text })}
```

Comment:
Would be really cool to define this callback as a class method !

for example:

```
export class AutocompleteSelector extends React.PureComponent {
  constructor(props) {
    super(props);
    this.state = {
      queryText: '',
    };
  }

onChangeText = text => ...

...


<SearchBar 
    ....
    onChangeText={this.onChangeText}
    ...
    />
```

Reply:
Is it a necessary class that you would like to implement on this one or we can discuss this later?

Reply:
This is not a new class, it is defining the callback as a class method - this makes the code easier to understand and also makes the function defined only once, reducing the number of re-renders